src/main/java/cn/nukkit/Player.java

```diff
@@ -307,6 +308,12 @@ public class Player extends EntityHuman implements CommandSender, InventoryHolde
     
     @PowerNukkitOnly
     @Since("1.4.0.0-PN")
+    private boolean showingCredits;
+    
+    @PowerNukkitOnly
+    @Since("1.4.0.0-PN")
+    private boolean hasSeenCredits;
+    
     public float getSoulSpeedMultiplier() {
```

Comment:
This commit has removed both `PowerNukkitOnly` and `Since` annotations from `getSoulSpeedMultiplier` 
```suggestion
    @PowerNukkitOnly
    @Since("FUTURE")
    private boolean showingCredits;

    @PowerNukkitOnly
    @Since("FUTURE")
    private boolean hasSeenCredits;
    
    @PowerNukkitOnly
    @Since("1.4.0.0-PN")
    public float getSoulSpeedMultiplier() 
    ```